publications-2018.bib

```diff
@@ -1258,8 +1280,9 @@ @phdthesis{phdthesis
 	author       = {Filippo Guido Davide Sacco},
 	title        = {A 3D adaptive boundary element method for potential flow with nonlinear Kutta condition},
 	school       = {Politecnico di Milano},
-	year         = {2017},
+	year         = {2018},
 	month        = {4}
+	url = {https://www.politesi.polimi.it/handle/10589/140106},
```

Comment:
bibtex is also going to complain about this line: you have a stray comma at the end of the line, and none on the previous one.